keyboards/adafruit/macropad/readme.md

```diff
@@ -0,0 +1,29 @@
+# Adafruit MacroPad RP2040
+
+![AdafruitMacropad](https://i.imgur.com/dSBSwcJ.jpeg)
```

Comment:
```suggestion
![AdafruitMacropad](https://i.imgur.com/dSBSwcJh.jpeg)
```

Reply:
Pardon my ignorance but why the "h"?

Reply:
No worries.  It's a neat little hack for imgur.  It renders a lower resolution version that is better for the readme pages. 

Though, double checking, probably not needed here.

Reply:
Whoopsie 😅 should I revert it? Or leave your suggestion?